src/app/App/styles.ts

```diff
@@ -0,0 +1,5 @@
+import styled from 'styled-components';
+
+export const AppWrapper = styled.div`
+  font-eight: 20px;
```

Comment:
please fix the typo. (font-weight).

Also, let's add default `font-family: Montserrat, sans-serif` style here. And add this font to the app (@import or <link> in index.html);

Reply:
It was added not for the real style, but to show how styled-components work, but will fix

Reply:
thank you 👍 

also, could you please add general font styles here, as mentioned here https://github.com/MecenatOrg/frontend/pull/8/files/2e78d0568188d2bd95f94788dcbb664bd3303857#r407224718

Reply:
As I saw it will be changed here https://github.com/MecenatOrg/frontend/pull/8/files

Reply:
the same here.

Reply:
Typo was fixed, and the font family will be added here https://github.com/MecenatOrg/frontend/pull/8/files